### PR TITLE
Add smear correction

### PR DESCRIPTION
**Pull request by mirca**
Let's go XD

### CODE REVIEW

File: kadenza/calibration.py [hunk 1]
```diff
@@ -86,6 +86,19 @@ def smear_frame(self, channel):
             frame[:, columns[idx]] = smear_values[idx]
         return frame
 
+    def get_smear_at_columns(self, columns, channel):
+        """
+        colnums : list-like of integers
+        """
```
Comment (barentsen) [inline kadenza/calibration.py]: My only comment is that the docs should say that the return values are in units ADU

**Review by barentsen — APPROVED** [review]

**Review by barentsen — APPROVED** [review]

File: kadenza/kadenza.py [hunk 2]
```diff
@@ -570,7 +583,10 @@ def kadenza_tpf_main(args=None):
     else:
         cflist = args.cadencefile_list[0]
     factory = TargetPixelFileFactory(cflist,
-                                     args.pixelmap_file[0])
+                                     args.pixelmap_file[0],
+                                     correct_smear=args.correct_smear)
+    print("correct_smear: {}".format(args.correct_smear))
```
Comment (barentsen) [inline kadenza/kadenza.py]: Let's get rid of the print statement =)

**Comment by mirca** [comment]
Done =)

**Comment by barentsen** [comment]
Awesome :+1:
I'm merging this!

**Pull request merged**